### PR TITLE
feat: fir 19080 system engine test improvement

### PR DESCRIPTION
**Pull request by aymeric-dispa**
Ticket: https://packboard.atlassian.net/browse/FIR-19080

### CODE REVIEW

File: FireboltDotNetSdk.Tests/Integration/SystemEngineTest.cs [hunk 1]
```diff
@@ -35,25 +32,25 @@ public void Cleanup()
...
+                {
+                    cursor.Execute($"DROP ENGINE IF EXISTS {newEngineName}");
+                }
+                catch (System.Exception) { }
                 try
                 {
-                    cursor.Execute($"DROP DATABASE IF EXISTS {DatabaseName}");
+                    cursor.Execute($"DROP DATABASE IF EXISTS {newDatabaseName}");
                 }
                 catch (FireboltException) { };
```
Comment (ptiurin) [inline FireboltDotNetSdk.Tests/Integration/SystemEngineTest.cs:48]: Should we fail if cleanup is not possible?

**Review by ptiurin — APPROVED** [review]
Just one question, otherwise LGTM.

File: FireboltDotNetSdk.Tests/Integration/SystemEngineTest.cs (hunk 1 above)
Comment (aymeric-dispa) [reply FireboltDotNetSdk.Tests/Integration/SystemEngineTest.cs:48]: I don't want to do assertions in the cleanup method as any failure would leave the method (could play with booleans but I want to keep it short and clean). Instead I will copy that line in a test and I'll keep the cleanup method as is

**Review by stepansergeevitch — APPROVED** [review]
lgtm

**Pull request merged**